build.sbt

```diff
@@ -79,11 +79,11 @@ lazy val macrosSetting = Seq(
   autoScalaLibrary := false,
 
   libraryDependencies ++= Seq(
-    "com.novocode" % "junit-interface" % "0.11" % "test",
+    // "com.novocode" % "junit-interface" % "0.11" % "test",
 
     // Dotty depends on stdlib 2.11.5, best use that too (0.1-SNAPSHOT is
     // actually 2.11.5, published under ch.epfl.lamp)
-    "ch.epfl.lamp" % "scala-library_2.11" % "0.1-SNAPSHOT",
+    // "ch.epfl.lamp" % "scala-library_2.11" % "0.1-SNAPSHOT",
```

Comment:
I'm not exactly sure why -- having it would cause problems about dependencies. The latest `sbt-dotty` also doesn't have this line.